sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubAsyncProducerTest.java

```diff
@@ -62,7 +62,7 @@ public void teardown() {
     }
 
     /**
-     * Verifies that sending multiple events will result in calling producer.send(List<Message>).
+     * Verifies that sending multiple events will result in calling {@code producer.send(List<Message>)}.
```

Comment:
This is a test. You don't need to worry about fixing up javadoc in test classes (add a suppression if necessary). We care about JavaDoc only in public API and samples.

Reply:
reverted and suppress added